src/main/java/org/jvnet/hudson/tools/versionnumber/VersionNumberBuilder.java

```diff
@@ -166,48 +195,48 @@ public String getEnvironmentPrefixVariable() {
         return this.environmentPrefixVariable;
     }
     private Run getPreviousBuildWithVersionNumber(AbstractBuild build) {
-        String envPrefix;
-        
-        if (this.environmentPrefixVariable != null) {
-            try {
-                EnvVars env = build.getEnvironment(null);
-                
-                envPrefix = env.get(this.environmentPrefixVariable);
-            } catch (IOException e) {
-                envPrefix = null;
-            } catch (InterruptedException e) {
-                envPrefix = null;
-            }
-        } else {
-            envPrefix = null;
-        }
-        
+        String envPrefix;
+        
+        if (this.environmentPrefixVariable != null) {
+            try {
+                EnvVars env = build.getEnvironment(null);
+                
+                envPrefix = env.get(this.environmentPrefixVariable);
+            } catch (IOException e) {
+                envPrefix = null;
+            } catch (InterruptedException e) {
+                envPrefix = null;
+            }
+        } else {
+            envPrefix = null;
+        }
+        
         // a build that fails early will not have a VersionNumberAction attached
         Run prevBuild = build.getPreviousBuild();
-        
+        
```

Comment:
please avoid empty-line changes. They make the review and merge operations more difficult